examples/python/aeotecdw2e.py

```diff
@@ -49,32 +50,32 @@ def exitHandler():
     sensor.optionsLock()
 
     # Next, initialize it.
-    print "Initializing, this may take awhile depending on your ZWave network"
+    print("Initializing, this may take awhile depending on your ZWave network")
 
     sensor.init(defaultDev)
-    print "Initialization complete"
+    print("Initialization complete")
 
-    print "Querying data..."
+    print("Querying data...")
 
     while (True):
         if (sensor.isDeviceAvailable()):
-            print "Alarm status:",
-            print sensor.isAlarmTripped()
+            print("Alarm status:", end=' ')
+            print(sensor.isAlarmTripped())
 
-            print "Tamper Switch status:",
-            print sensor.isTamperTripped()
+            print("Tamper Switch status:", end=' ')
+            print(sensor.isTamperTripped())
 
-            print "Battery Level:",
-            print sensor.getBatteryLevel(),
-            print "%"
+            print("Battery Level:", end=' ')
+            print(sensor.getBatteryLevel(), end=' ')
+            print("%")
 
-            print
+            print()
         else:
-            print "Device has not yet responded to probe."
-            print "Try waking it, or wait until it wakes itself if ",
-            print "configured to do so."
+            print("Device has not yet responded to probe.")
+            print("Try waking it, or wait until it wakes itself if ", end=' ')
```

Comment:
Oh, and the reason it was written that way originally was to avoid line wrapping (this is/was a requirement).